doc/requirements.txt

```diff
@@ -3,6 +3,6 @@ ipython
 matplotlib
 nbsphinx
 sphinx-argparse
-sphinx>=2.1.2
+sphinx>=2.1.2,<3
```

Comment:
@effigies - i fixed a sphinx related bug on pydra a couple of days back. if its the same issue, perhaps we can apply that here as well. it had to do with a `:noindex:` warning becoming an error.

Reply:
I think the issue is with some custom Sphinx extensions using a deprecated API. In any event, I'd prefer to fix that separately. If we do before this is merged, I can rebase this commit away.